content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -315,6 +315,10 @@ tolerations to all daemons, to prevent DaemonSets from breaking.
 Adding these tolerations ensures backward compatibility. You can also add
 arbitrary tolerations to DaemonSets.
 
+{{< note >}}
+If the `nodeName` for a Pod manifest is manually specified, the scheduler will assign the Pod to the node specified, regardless of any taints or tolerations.
```

Comment:
If you reword this, I recommend you [write in the active voice](https://kubernetes.io/docs/contribute/style/style-guide/#use-active-voice).

Reply:
Lovely, thanks for the review @sftim. I will modify it to the below:
```markdown
{{< note >}}
If you manually specify the `nodeName` for a Pod manifest, the scheduler will assign the Pod to the specified node, regardless of any taints or tolerations.
{{< /note >}}
```

I also want to suggest that it appears under the heading "Concepts". The example in the heading was what sprung the PR in the first place. What do you think?

Reply:
You're still using a `note` (I wouldn't), and you're still mentioning the scheduler assigning the Pod to a node (which in this case is not going to happen; the scheduler only ever binds _unbound_ Pods to nodes).

The move sounds OK.

Reply:
I get it now. 
If that's so, adding it just above the statement "Here's an example of a pod that uses tolerations:" without a note makes sense because developers have just learned how to prevent pods from being scheduled to inappropriate nodes using taints and tolerations. However, they may be unaware of how a previous factor, `nodeName`, which they learned earlier in the [Assigning Pods to Nodes](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/) page can influence this new configuration.

The final statement becomes:
**However, if you manually specify the `nodeName` for a Pod manifest without specifying a matching toleration, the kubelet on the named tainted node tries to place the Pod on that node, regardless of any taints or tolerations.**
 
**Here's an example of a pod that uses tolerations:**

What do you think of the final statement and also where it will be newly placed compared to after the section 'Concept'?
In the final statement, I also thought about using "in the PodSpec" instead of "for a Pod manifest" since PodSpec was used on the page.

Reply:
Try this:
```markdown
The default Kubernetes scheduler takes taints and tolerations into account when
selecting a node to run a particular Pod. However, if you manually specify the
`.spec.nodeName` for a Pod, that action bypasses the scheduler; the Pod is then
bound onto on the node where you assigned it, even if there are `NoSchedule`
taints on that node that you selected.
If this happens and the node also has a `NoExecute` taint set, the kubelet will
eject the Pod unless there is an appropriate toleration set.

Here's an example of a pod that has some tolerations defined:
```

Reply:
I appreciate your review @sftim, thank you!